api/envoy/extensions/filters/http/ratelimit/v3/rate_limit.proto

```diff
@@ -388,11 +171,24 @@ message RateLimitPerRoute {
   // [#not-implemented-hide:]
   OverrideOptions override_option = 2 [(validate.rules).enum = {defined_only: true}];
 
-  // Rate limit configuration. If not set, uses the
+  // Rate limit configuration that is used to generate a list of descriptor entries based on
+  // the request context. The generated entries will be used to find one or multiple matched rate
+  // limit rule from the ``descriptors``.
+  // If this is set, then
   // :ref:`VirtualHost.rate_limits<envoy_v3_api_field_config.route.v3.VirtualHost.rate_limits>` or
-  // :ref:`RouteAction.rate_limits<envoy_v3_api_field_config.route.v3.RouteAction.rate_limits>` fields instead.
-  // [#not-implemented-hide:]
-  repeated RateLimitConfig rate_limits = 3;
+  // :ref:`RouteAction.rate_limits<envoy_v3_api_field_config.route.v3.RouteAction.rate_limits>` fields
+  // will be ignored.
+  //
+  // .. note::
+  //   Not all configuration fields of
+  //   :ref:`rate limit config <envoy_v3_api_msg_config.route.v3.RateLimit>` is supported at here.
+  //   Following fields are not supported:
+  //
+  //   1. :ref:`rate limit stage <envoy_v3_api_field_config.route.v3.RateLimit.stage>`.
+  //   2. :ref:`dynamic metadata <envoy_v3_api_field_config.route.v3.RateLimit.Action.dynamic_metadata>`.
+  //   3. :ref:`disable_key <envoy_v3_api_field_config.route.v3.RateLimit.disable_key>`.
+  //   4. :ref:`override limit <envoy_v3_api_field_config.route.v3.RateLimit.limit>`.
+  repeated config.route.v3.RateLimit rate_limits = 3;
```

Comment:
This is a break change to the API to avoid the redundant RateLimitConfig message. But this field is never be implemented and is only extension API, so, I think this should be fine. cc @envoyproxy/api-shepherds

Reply:
If never implemented sounds good.